script.js

```diff
@@ -6,37 +6,38 @@ const navigationMenu = document.querySelector(".navigation__menu");
 const body = document.querySelector("body");
 const cart = document.querySelector('.navigation__cart');
 const addToCart = document.querySelector('.cart');
+const nextNavigationButton = document.querySelector('.navigation__button--features');
+const backNavigationButton = document.querySelector('.sub-navigation__button--back');
+const backNavigationWord = document.querySelector('.sub-navigation__text--back');
+const subMenu = document.querySelector('.sub-navigation');
 
 hamburger.addEventListener('click', activeSlide);
-
-main.onclick = () => {
-   hamburger.classList.remove("active");
-   navigationMenu.classList.remove("active");
-   addToCart.classList.remove("active");
-   body.classList.remove("fixed-body");
-}
-
-footer.onclick = () => {
-    hamburger.classList.remove("active");
-    navigationMenu.classList.remove("active");
-    addToCart.classList.remove("active");
-    body.classList.remove("fixed-body");
-}
-
 cart.addEventListener('click', openCart);
 
 function activeSlide() {
+    if (subMenu.classList.contains("active")) {
+        closeAll();
+    } else {
     hamburger.classList.toggle("active");
     navigationMenu.classList.toggle("active");
     addToCart.classList.remove("active");
     fixedBody();
+    }
+}
+
+function closeAll() {
+    hamburger.classList.remove("active");
+    navigationMenu.classList.remove("active");
+    subMenu.classList.remove("active");
+    body.classList.remove("fixed-body");
 }
 
 function openCart() {
     addToCart.classList.toggle("active");
     navigationMenu.classList.remove("active");
     hamburger.classList.remove("active");
     body.classList.remove("fixed-body");
+    subMenu.classList.remove("active");
 }
 
 function fixedBody() {
```

Comment:
Is this function for scroll enabling  when menu/sub-menu is closed  ? 
If yes, maybe try to use naming enableScroll? In this case people will better understand what you're doing inside specific function